design-proposals/cluster-orch-addon.md

```diff
@@ -0,0 +1,88 @@
+
+| Addon                      | Default in 3.0 | Default in 3.1 (EMF-Managed) | Default in 3.1 (EMT-S) | Notes                                                                   |
+| -------------------------- | -------------- | ---------------------------- | ---------------------- | ----------------------------------------------------------------------- |
+| gatekeeper and constraints | Y              | N                            | N                      | Replace with Kubernetes Pod Security Admission Controller and Standards |
+| cert-manager               | Y              | N                            | N                      | Move to optional observability package                                  |
+| telegraf                   | Y              | N                            | N                      | Move to optional observability package                                  |
+| prometheus stack           | Y              | N                            | N                      | Move to optional observability package                                  |
+| node-exporter              | Y              | N                            | N                      | Move to optional observability package                                  |
+| openebs                    | Y              | N                            | N                      | Move to optional package                                                |
+| fluent-bit                 | Y              | N                            | N                      | Move to optional package                                                |
+| nfd                        | Y              | Y                            | Y                      |                                                                         |
+| network-policies           | Y              | Y                            | Y                      |                                                                         |
+| local-path-provisioner     | N              | Y                            | Y                      | Built into K3s                                                          |
+| kube-metrics               | N              | N                            | Y                      | Built into K3s                                                          |
+| traefik ingress controller | N              | N                            | Y                      | Built into K3s                                                          |
+| serviceLB                  | N              | N                            | Y                      | Build into K3s                                                          |
+| kubernetes-dashboard       | Y (EMT-S only) | N                            | Y                      |                                                                         |
```

Comment:
@hyunsun  i think we need to mention Calico as part of this list. 

Can we clarify the k3s images already has coredns. Do we plan to include external etcd ?

It is also better to add the second profile we are supporting additional K3s ext – Kubevirt (MF fork), metallb, GPU device plugin and MF device plugin.

Reply:
Added calico. coredns is included. No external etcd. Let me add the second profile next week once we get more clarity on the execution plan.